cypress/support/pages/settings.pageObject.js

```diff
@@ -0,0 +1,63 @@
+import PageObject from '../PageObject';
+
+class SettingsPageObject extends PageObject {
+  url = '/settings';
+
+  get usernameField() {
+    return cy.getByDataCy('settings-username-field');
+  }
+
+  editUsername() {
+    this.usernameField.clear().type(username);
+  }
+
+  get bioField() {
+    return cy.getByDataCy('settings-bio-field');
+  }
+
+  editBio() {
+    this.bioField.clear().type(bio);
+  }
+
+  get emailField() {
+    return cy.getByDataCy('settings-email-field');
+  }
+
+  editEmail() {
+    this.emailField.clear().type(email);
```

Comment:
The `editEmail` method is missing the `email` parameter. You should pass `email` as an argument to this method to ensure it works correctly.